code/js/rcalc-localization.js

```diff
@@ -0,0 +1,25 @@
+var set_locale_to = function(locale) {
+  if (locale) {
+    $.i18n().locale = locale;
+  }
+  $('title').i18n();
+  $('body').i18n();
+};
+
+jQuery(function() {
+  $.i18n().load({
+    'en': '../js/i18n/en.json',
+    'cs': '../js/i18n/cs.json'
+  }).done(function() {
+    set_locale_to(url('?locale'));
+
+    History.Adapter.bind(window, 'statechange', function() {
+      set_locale_to(url('?locale'));
+    });
+
+    $('.rcalc-dropdown-content-language').on('click', 'a', function(e) {
```

Comment:
Why is not in the constant file? You created spaghetti code. This is not good.

Reply:
created two constants, one special character "." as const DOT an class const RCALC-DROPDOWN-CONTENT-LANGUAGE the result is (DOT + RCALC-DROPDOWN-CONTENT-LANGUAGE)